### PR TITLE
Fix AWS CLI installation instructions in KPO doc

### LINKED ISSUE #2575: Update AWS CLI installation instructions for V2
### Content type

Airflow concepts

### Request type

Additional information for an existing guide or tutorial

### Describe your Learn request.

Current instructions for installing AWS CLI will install V1, but they should be updated to install V2. More info in Slack thread below. 
Affects: https://docs.astronomer.io/learn/kubepod-operator

### Desired publish date

_No response_

### Airflow Version

_No response_

### Reviewer

_No response_

### Additional information

https://astronomer.slack.com/archives/C015V2JFKT5/p1689617459261229

### PR DESCRIPTION
**Pull request by TJaniF**
This just came up in the docs channel but also would resolve #2575 I think :)

### CODE REVIEW

File: astro/kubernetespodoperator.md [hunk 1]
```diff
@@ -427,7 +427,31 @@ Astronomer recommends creating a Kubernetes cluster connection because it's more
...
+### Step 4: Install the AWS CLI in your Astro environment
+
+In order to be able to connect to your external EKS cluster you need to install the AWS CLI in your Astro project. 
+
+1. Add the following to your `Dockerfile` to install the AWS CLI:
+
+    ```dockerfile
+    USER root
+
+    RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip" -o "awscliv2.zip"
```
[the remark this answers is not in the document]
Comment (TJaniF) [reply astro/kubernetespodoperator.md:439]: 👍 thank you!

Comment (TJaniF) [inline astro/kubernetespodoperator.md:439]:
```suggestion
    RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
    # Note: if you are testing your pipeline locally you may need to adjust the zip version to your dev local environment
```

**Review by kentdanas — APPROVED** [review]

File: astro/kubernetespodoperator.md [hunk 2]
```diff
@@ -427,7 +427,32 @@ Astronomer recommends creating a Kubernetes cluster connection because it's more
 
 You can now specify this connection in the configuration of any KubernetesPodOperator task that needs to access your external cluster. 
 
-### Step 4: Configure your task
+### Step 4: Install the AWS CLI in your Astro environment
+
+In order to be able to connect to your external EKS cluster you need to install the AWS CLI in your Astro project.
```
Comment (lzdanski) [inline astro/kubernetespodoperator.md:432]:
```suggestion
To connect to your external EKS cluster, you need to install the AWS CLI in your Astro project. 
```

**Review by lzdanski — APPROVED** [review]
Teeny wording suggestion, otherwise LGTM

**Pull request merged**